specification/postgresql/resource-manager/Microsoft.DBforPostgreSQL/preview/2020-02-14-privatepreview/examples/ServerCreate.json

```diff
@@ -0,0 +1,90 @@
+{
+  "parameters": {
+    "serverName": "pgtestsvc4",
+    "resourceGroupName": "TestGroup",
+    "api-version": "2020-02-14-privatepreview",
+    "subscriptionId": "ffffffff-ffff-ffff-ffff-ffffffffffff",
+    "parameters": {
+      "location": "westus",
+      "properties": {
+        "administratorLogin": "cloudsa",
+        "administratorLoginPassword": "password",
+        "serverEdition": "GeneralPurpose",
+        "vCores": 4,
+        "storageQuotaInMb": 524288,
+        "version": "12",
+        "standbyCount": 0,
+        "backupRetentionDays": 7,
+        "vnetInjArgs": {
+          "delegatedSubnetName": "autobot-subnet-pg-ss",
```

Comment:
remove autobot from delegated subnet name and vnet name

Reply:
its just name of the subnet name. it is a sample.i dont think it is an issue.